### PR TITLE
Implement post deletion endpoint

### PR DESCRIPTION
**Pull request by mccodav1**

### CODE REVIEW

File: PosteAPI/urls.py
Comment (fmehdad97) [inline PosteAPI/urls.py:1]: Thank you for formatting the URLS!

File: PosteAPI/views.py
Comment (fmehdad97) [inline PosteAPI/views.py:1]: Seems to do what it needs to do.

**Review by fmehdad97 — APPROVED** [review]
Would have been nice to have the swagger auto schema, but I guess we can add that in towards the end of development.

**Comment by mccodav1** [comment]
> Would have been nice to have the swagger auto schema, but I guess we can add that in towards the end of development.

I'll add this as a User Story. I agree having API documentation as we go is very nice to have, and Swagger assists with testing / troubleshooting.

**Pull request merged**